src/shell/lisa_shell

```diff
@@ -256,7 +256,7 @@ EOF
 
 function _lisa-test {
 CLASS=${1}
-nosetests -v tests/$CLASS
+nosetests --nocapture --nologcapture -v tests/$CLASS
```

Comment:
Actually while running this code on a (misconfiguration) target to test it, I realized that this feature can be counter productive in case we have multiple failures. For example, if the testenv setup fails for whatever reasons, usually we split a WARNING statement explaining the reason for the failure right before raising the exception.
Unfortunately,  since with this patch all the test failures are grouped at the end, it becomes difficult to correlate an error with its explanation.

I would propose to remove this patch from this PR and post-pone the logs clean-up into a separate PR once we investigated a better solution.